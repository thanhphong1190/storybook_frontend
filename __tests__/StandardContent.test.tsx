import {
  StandardContent,
  PageContentProps
} from '../src/components/standard-content/StandardContent'
import renderer from 'react-test-renderer'

const props: PageContentProps = {
  paragraphs: [
    {
      type: 'title',
      text: 'More Payments & Shopping Icons'
    },
    {
      type: 'text',
      text:
        'Pictures, abstract symbols, materials, and colors are among the ingredients with which a designer or engineer works. To design is to discover relationships and to make arrangements and rearrangements among these ingredients.'
    },
    { type: 'space' },

    {
      type: 'title',
      text: 'More Payments & Shopping Icons'
    },
    {
      type: 'text',
      text:
        'Pictures, abstract symbols, materials, and colors are among the ingredients with which a designer or engineer works. To design is to discover relationships and to make arrangements and rearrangements among these ingredients.'
    }
  ]
}

it('StandardContent renders correctly', () => {
  const tree = renderer.create(<StandardContent {...props} />).toJSON()
  expect(tree).toMatchSnapshot()
})
