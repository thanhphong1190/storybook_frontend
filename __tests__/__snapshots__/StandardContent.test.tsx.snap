// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StandardContent renders correctly 1`] = `
<div
  className="StandardContentstyles__Container-sc-2x20s6-0 fjzDWf"
>
  <div
    className="globalstyles__Text-sc-1nc69wi-9 jrNbvV"
    fontWeight="semibold"
    size="lg"
  >
    More Payments & Shopping Icons
  </div>
  <div
    className="globalstyles__Text-sc-1nc69wi-9 hVBppN"
  >
    Pictures, abstract symbols, materials, and colors are among the ingredients with which a designer or engineer works. To design is to discover relationships and to make arrangements and rearrangements among these ingredients.
  </div>
  <div
    className="globalstyles__Space-sc-1nc69wi-1 QmALU"
    height={2}
  />
  <div
    className="globalstyles__Text-sc-1nc69wi-9 jrNbvV"
    fontWeight="semibold"
    size="lg"
  >
    More Payments & Shopping Icons
  </div>
  <div
    className="globalstyles__Text-sc-1nc69wi-9 hVBppN"
  >
    Pictures, abstract symbols, materials, and colors are among the ingredients with which a designer or engineer works. To design is to discover relationships and to make arrangements and rearrangements among these ingredients.
  </div>
</div>
`;
